Sources/App/Views/PackageController/PackageShowView.swift

```diff
@@ -0,0 +1,118 @@
+import Plot
+
+
+class PackageShowView: PublicPage {
+
+    let package: Package
+
+    init(_ package: Package) {
+        self.package = package
```

Comment:
I guess here's where we interface with our view model. After tackling #62 - depending on how that goes 😅 - shall I start filling in the query and plug it in here?

Reply:
Yes, that'd be wonderful. 👍

My gut feeling says that the view model (for example) shouldn't construct the full sentences that are going to be used, but instead populate everything that might be needed. 

Maybe something like this:

To construct this sentence:

"In development for over 5 years, with 1,433 commits and 79 releases."

The view model could contain:

* First commit date (`Date`)
* Number of commits (`Int`)
* Number of releases (`Int`)

and to construct this sentence:

"By Christian Noon, Mattt, Jon Shier, Kevin Harwood, and 186 other contributors."

* Array of:
  * Full Name (`String`)
  * Profile URL (`URL`)
* Number of other contributors (`Int`)

Is that what you were thinking?

Reply:
Actually, the place for this discussion isn't a merged PR. I'm going to make a new issue for this so we can discuss it and track it.